README.md

```diff
@@ -17,6 +17,22 @@ There is an awesome library [mjml](https://mjml.io/) with github repo here [http
 `MJML` is a markup language created by [Mailjet](https://www.mailjet.com/).
 So in order to create emails on the fly we created a library with `React` components.
 
+## What's new in V3?
```

Comment:
I wonder if we should mention that v2 is supposed to be a perfect replica of wix's mjml-react?

## Get started today with V2 - 100% backwards compatible with https://github.com/wix-incubator/mjml-react/

V2 is a drop in replacement for https://github.com/wix-incubator/mjml-react/, with some additional support. If there's a component missing you can make a PR against https://github.com/Faire/mjml-react/tree/main-v2 or submit an issue and we'll try to unblock you.